recipes/self-managed-otlp-ingest/traffic/hey.Dockerfile

```diff
@@ -0,0 +1,24 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM golang:1.21-alpine3.19 AS build
+
+RUN apk add --no-cache wget
+RUN wget -O /hey https://hey-release.s3.us-east-2.amazonaws.com/hey_linux_amd64
```

Comment:
how did you end up with this command and URL?  Seems odd to be downloading something from AWS.  Should we update our samples to do this as well?  https://github.com/GoogleCloudPlatform/opentelemetry-operations-java/blob/404b12a72ca8e13b0b4809a476f31941bc0e5b60/examples/instrumentation-quickstart/docker-compose.yaml#L48

Reply:
Links from here - https://github.com/rakyll/hey?tab=readme-ov-file#installation. 

I'm not a big fan of docker compose, since it seems to abstract too much - it makes the deployment simple, but makes it difficult to understand the sample IMO.

If the link is the only problem, I could alternatively build from source.

Reply:
Can you just add a comment linking to the README as the source of the download link?